fighters/trail/src/opff.rs

```diff
@@ -240,7 +240,25 @@ unsafe fn flower_frame(boma: &mut BattleObjectModuleAccessor, status_kind: i32)
         }
     }
 }
-    
+
+// properly cycles Sora's HUD to fire in training mode on reset
+unsafe fn training_cycle(fighter: &mut L2CFighterCommon, boma: &mut BattleObjectModuleAccessor, frame: f32) { 
+    if VarModule::is_flag(boma.object(), vars::trail::instance::CYCLE_MAGIC)
```

Comment:
Please swap these two lines so we can lessen the amount of checks being handled

Reply:
yes sir